jetty-http/src/main/java/org/eclipse/jetty/http/CompressedContentFormat.java

```diff
@@ -32,9 +45,9 @@
 
     public CompressedContentFormat(String encoding, String extension)
     {
-        _encoding = encoding;
-        _extension = extension;
-        _etag = "--" + encoding;
+        _encoding = StringUtil.asciiToLowerCase(encoding);
+        _extension = StringUtil.asciiToLowerCase(extension);
+        _etag = ETAG_SEPARATOR + encoding;
```

Comment:
Don't we want here the lower-cased `_encoding` to be appended?

Reply:
yes - using lowercase greatly simplifies the code in equals, which actually had a bug if _encoding was null but ccf._encoding was not null